joblib/test/data/generate_data.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+for i in py26-np16 py27-np17 py33-np18 py34-np19 py35-np19
+do 
+  . activate $i
+  python create_numpy_pickle.py
+  . deactivate
```

Comment:
I don't think you need the deactivate by the way (at least not in conda environments)